src/main/java/com/google/cloud/spanner/pgadapter/statements/CopyStatement.java

```diff
@@ -188,6 +188,11 @@ private void queryInformationSchema() throws SQLException {
     }
   }
 
+  @Override
+  public void handleExecutionException(SQLException e) {
+    super.handleExecutionException(e);
+  }
```

Comment:
nit: this method can be removed in its entirety, as it only calls the super implementation

Reply:
The super version is protected and not public.